specs/fulltext_update.proto

```diff
@@ -8,5 +8,5 @@ message FulltextUpdate {
     Status status = 3;
     string acknowledgements = 4;
     repeated string dataset = 5;
+	  repeated string facilities = 6;
```

Comment:
There seems to be a mix of tab and spaces for indentation, could you please use spaces to be consistent with the rest?

Reply:
Fixed! It looked correct in my editor (atom) but I changed my setting so I'll be able to catch this next time.